src/routes/stats.test.js

```diff
@@ -0,0 +1,31 @@
+import express from 'express';
+import request from 'supertest';
+import passport from 'passport';
+import AnonymousStrategy from 'passport-anonymous';
+
+import stats from './stats';
+import database from '../modules/database';
+
+/* eslint-disable camelcase */
```

Comment:
What's the error here?

Reply:
Sorry, didn't see this until I'd already merged. I don't remember, it is in most of the route tests. It may have been residual from something a while back. I don't see anything that would require it now.

Reply:
Sure thing, I'll test a branch taking them out where I can.